src/components/Lend/LoanSearch/ChallengeCallout.vue

```diff
@@ -108,59 +67,63 @@ export default {
 			type: String,
 			default: undefined,
 		},
-		teamId: {
-			type: String,
-			default: ''
-		}
 	},
 	data() {
 		return {
 			mdiArrowTopRight,
 		};
 	},
 	computed: {
-		shareLenderName() {
-			return this.shareLender?.name ?? '';
-		},
-		shareLenderImage() {
-			return this.shareLender?.image?.url ?? '';
-		},
-		headerCallout() {
-			return this.shareLenderName
-				? `Support ${this.shareLenderName} and help `
-				: 'Help ';
-		},
 		participants() {
-			return (this.goalParticipationForLoan ?? [])
+			const rawParticipants = (this.goalParticipationForLoan ?? [])
 				.filter(l => l?.lender?.id !== this.currentLender?.lender?.id)
 				.concat(this.currentLender)
 				.map(p => ({ ...p?.lender, image: { url: p?.lender?.image?.url ?? '' } }))
 				.slice(0, 3);
+
+			return {
+				totalCount: rawParticipants.length,
```

Comment:
I guess this query has a total count field, in other way as slicing the array, this is returning 3 as a max number, true?